indicators/talib_test.go

```diff
@@ -0,0 +1,771 @@
+// 	expectedLows := []float64{197.64, 195.78, 197.35, 199.89, 200.12, 198.55, 197.99, 195.61, 195, 194.75, 197.97, 197.43, 199.87, 200.83, 200.57, 198.64, 196.09, 196.33, 195.42, 196.01, 198.9, 199.8, 200.72, 201.39, 200.49, 201.72, 202.43, 203.69, 204.84, 205.17, 205.42, 205.18, 205.24, 206.68, 206.85, 207.35, 207.11, 206.4, 206.52, 206.85, 205.98, 206.2, 203.3, 203.54, 200.84, 200.37, 201.11, 201.67, 202.53, 203.49, 203.2, 205.26, 206.39, 206.67, 205.51, 202.5, 201.71, 201.88, 203.7, 203.16, 202.44, 202.36, 202.12, 204, 204.26, 204.49, 205.89, 205.78, 205.54, 206.72, 206.7, 204.66, 205.75, 206.28, 206.68, 206.82, 208.3, 207.43, 207.4, 207.04, 205.16, 206.08, 207.88, 205.59, 204.74, 204.63, 207.54, 207.27, 206.29, 206.69, 207.89, 209.07, 208.88, 209.65, 209.51, 209.34, 209.62, 207.36, 207.9, 208.97, 207.79, 208.22, 207.68, 208.56, 206.8, 206.45, 205.18, 205.15, 206.05, 208.13, 206.68, 205.33, 205.62, 207.25, 207.96, 208.48, 209.55, 209.71, 208.18, 207.54, 207.5, 203.15, 203.57, 205.21, 205.03, 203.49, 204.67, 202.27, 202.63, 205, 206.68, 207.4, 208.28, 209.53, 209.94, 210.24, 209.42, 208.6, 207.86, 205.7, 204.5, 205.49, 207.16, 207.84, 208.17, 207.47, 207.06, 207.75, 206.05, 205.65, 206.97, 206.35, 204.82, 206.35, 206.13, 206.4, 207.66, 206.02, 201.71, 195.64, 185.42, 185.2, 189.96, 194.84, 196.31, 195.48, 189.65, 192.47, 193.39, 190.46, 193.77, 192.64, 192.41, 193.22, 193.84, 194.44, 196.62, 197.52, 194.29, 195.28, 192.73, 192.45, 191.01, 191.73, 186.9, 187.01, 189.24, 190.94, 188.65, 195.3, 196.62, 197.72, 197.77, 200.14, 200.23, 199.07, 198.11, 198.9, 201.63, 201.3, 201.65, 200.66, 201.78, 206.02, 205.78, 204.98, 205.78, 207.12, 206.7, 207.09, 208.73, 209.12, 208.91, 208.5, 206.85, 206.28, 206.51, 203.63, 201.34, 201.12, 204.25, 204.82, 207.32, 208.07, 207.83, 206.64, 208.08, 208.19, 207.46, 208.2, 207.3, 204.39, 204.39, 207.12, 205.27, 204.13, 204.2, 200.69, 200.87, 203.49, 205.15, 203.65, 200.02, 201.41, 202.72, 204.69, 205.68, 204.86, 206.51, 205.93, 203.87}
+//
+// 	resultHigh, resultOpen, resultClose, resultLow := HeikinashiCandles(testHigh, testOpen, testClose, testLow)
+// 	for i := range expectedHighs {
+// 		if resultHigh[i] != expectedHighs[i] {
+// 			t.Errorf("Highs error: Expected %f at cell %d got %f ", expectedHighs[i], i, resultHigh[i])
+// 		}
+// 	}
+//
+// 	for i, expected := range expectedOpens {
+// 		if resultOpen[i] != expected {
+// 			t.Errorf("Opens error: Expected %f at cell %d got %f ", expected, i, resultOpen[i])
+// 		}
+// 	}
+//
+// 	for i, expected := range expectedCloses {
+// 		if resultClose[i] != expected {
+// 			t.Errorf("Closes error: Expected %f at cell %d got %f ", expected, i, resultClose[i])
+// 		}
+// 	}
+//
+// 	for i, expected := range expectedLows {
+// 		if resultLow[i] != expected {
+// 			t.Errorf("Lows error: Expected %f at cell %d got %f ", expected, i, resultLow[i])
+// 		}
+// 	}
+// }
```

Comment:
reinstate?

Reply:
:+1: forgot to update with new data will do that now